torch/_inductor/codegen/cpp.py

```diff
@@ -3436,7 +3445,10 @@ def is_lowp_fp_scheduler(self, scheduler_node: SchedulerNode):
             return True
         # Propagate the dtype to check if all the fx node is bf16/fp16
         DataTypePropagation.propagate_scheduler_node(scheduler_node)
-        return get_loop_body_lowp_fp(scheduler_node._body) is not None
+        return (
+            get_loop_body_lowp_fp(scheduler_node._body)[0] is not None
+            and not get_loop_body_lowp_fp(scheduler_node._body)[1]
+        )
```

Comment:
I guess we do not this extra check with the "full-bit" support in this PR? The original motivation of having this check is that we want to use "full-bit" as much as possible. Now we use "full-bit" by default. We can check whether to add type conversion inside the legalization logic not before it. We can do this cleanup in the next PR.

Reply:
Yes, we can check whether to add type conversion inside the legalization logic. Will do this in the subsequent PRs.